### ISSUE TITLE
[Bug] Transition "Cut" is actually default transition.

### ISSUE THREAD

**Issue by gohamstergo** [bitfocus/companion-module-obs-studio#62]
"obs: Transition Preview to Program - Transition to use: Cut" doesnt actually do a Cut, it just engages the default transition that is set in OBS under "Scene Transitions." Should rename the current action to "Transition" (since thats what the button in OBS is labeled), and create a new button that engages the "Cut" transition specifically.

Currently, to initiate a Cut, you have to have 2 actions on the button: "Change Transition Type" to Cut, then also select Cut under "Transition P to P"

**Comment by bryce-seifert** [comment]
What version of OBS/websocket + platform are you experiencing this on. Trying to recreate this, but haven't been able to yet–it seems to be working properly for me on macOS 26.1.2

**Comment by gohamstergo** [comment]
my bad:
OBS 26.1.1
websocket 4.9

On Windows 10. So, you can set the scene transition to something like Fade, then click Cut in Companion and it cuts? I find it just does whatever transition I have selected on OBS.

**Comment by bryce-seifert** [comment]
Yeah, no matter what I change the transition in OBS to it seems to switch back to cut when I used the Companion action.

Also what version of Companion are you using, one of the latest betas?

**Comment by gohamstergo** [comment]
No beta, just 2.1.2.

**Comment by bryce-seifert** [comment]
I would try one of the 2.2.0 betas if you have a chance  just to see if that resolves it, there have been some changes including to the web socket version used since then.

**Comment by gohamstergo** [comment]
Updated to 2.2.0-53dfc78-2881 but behavior is the same. Is there a way to see the raw command being sent somewhere in the logs? All I see in Companion log is button pressed/released.

**Comment by bryce-seifert** [comment]
Ok, I think I figured this one out. 

So it seems like the "default" on the dropdown menu was set to 'null'. So even though the first transition in the list, likely Cut, was shown as selected in the dropdown, it wasn't sending a transition name, thus sending the default. If you were to select a different transition on the list, and then select Cut again it would work, which is why I had trouble recreating it at first.

I've updated this to have the dropdown to start with "Default" value at first, and added some checks to the action so it hopefully behaves as expected.
